ddp-workspace/projects/ddp-dsm-ui/src/app/WELCOME/auth/auth.component.ts

```diff
@@ -43,12 +44,24 @@ export class AuthComponent implements OnInit {
   popUpIsShown = true;
   authError: Observable<string | null>;
   pickList$: Observable<any>;
+  private source$: Observable<StorageEvent>;
 
   constructor(private auth: Auth, private title: Title, private router: Router) {}
 
   ngOnInit(): void {
+
+    this.source$ = fromEvent<StorageEvent>(window, 'storage');
+    this.source$.subscribe(
+      event => {
```

Comment:
Listen for storage events to see if the DSM_TOKEN_NAME is set to null.  This is how we are able to detect a logout event from another tab.